deploy/csi-azuredisk-node.yaml

```diff
@@ -83,6 +83,7 @@ spec:
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
             - "--metrics-address=0.0.0.0:29605"
+            - "--enable-perf-optimization=true"
```

Comment:
managed Azure Disk CSI driver on AKS does not support such customization for different clusters, so it's better put all sku configs as a map in go file, by that way, don't need to read & parse a config map, and we could leave this option to user setting (`enablePerfOptimization`) in storage class.

Reply:
You need to take this as a feature flag. We are not asking users to tweak this setting on day to day operations, this is our may day option, so I don't see any harm in keeping this. driver is used outside of AKS, where users will be able to mitigate if it is needed.